content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -0,0 +1,145 @@
+
+## Prerequisites 
+
+The minimum Agent version required is `7.50` or higher.
+
+## Device profile onboarding
+
+### Profile home page
+
+The [profile home][4] page is where you can see a snapshot of the device profiles you created using the [Device Onboarding Experience](#build-a-device-profile). 
+
+**Need updated screen shot** - This is a placeholder
+
+{{< img src="/network_device_monitoring/profile_onboarding/profile_home_page.png" alt="The Network Device profile home page" style="width:100%;">}}
+
+### Profile details
+
+1. Build your own NDM profile by navigating to [Infrastructure > Network Devices > Configuration][1]. 
+2. Click on SNMP Profiles > **Create New Profile**.
+
+{{< img src="/network_device_monitoring/profile_onboarding/create_profile.png" alt="The Network Device profile creation page" style="width:100%;">}}
+
+3. Provide your device profile a name, vendor information(optional) and description (optional).
+4. Select the `SysObjectID`. This is what is used to match network devices to the device profiles that define what is collected and monitored from each device. 
+
+{{< img src="/network_device_monitoring/profile_onboarding/Sys_object_ID_Field.png" alt="The Network Device profile creation page showing the Sys Object ID Dropdown" style="width:100%;">}}
+
+### Global Tags
+
+Add global tags for more advanced and granular options, which allows you to assign a weight to a specific metric.
+
+{{< img src="/network_device_monitoring/profile_onboarding/Add_global_tags.png" alt="The Network Device profile creation page showing the Add Global Tags dropdown" style="width:100%;">}}
+
+#### Advanced Options
+
+| Modification       | Description   |
+| ------------- | ------------- |
+| No Modification | The device's returned value will be used directly as the tag value. |
+| Format | This can be [mac_address][5] or [ip_address][6]. |
+| Extract Value | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |
+| Mapping | This is described [here][8]. Note: our current docs may imply this can ONLY be used for table tags, which needs correction. |
+
+
+### Scalar metrics
+
+See [Metrics Advanced Options](#metrics-advanced-options).
+
+### Tabular metrics
+
+See [Metrics Advanced Options](#metrics-advanced-options).
+
+### Add Tabular Tags
+
+Adding tags to tabular metrics is similar to adding global tags, with two additional options:
+
+1. Select whether the tag value originates from an `OID` value or a segment of the table index. If `Index` is chosen as the source, an index position must be specified, which then becomes the tag.
+
+<details>
```

Comment:
I really like how this looks! I wonder if this can also indent so it's in line with number 1.